### DISCUSSION TITLE
How to Flush SLOG?

### DISCUSSION THREAD

**Discussion by spflanze** [openzfs/zfs#14449] [Q&A] [answered]
The need is for a BASH script to command the "Separate intent LOG SSD" (SLOG), which is set up to cache writes to the slower HDD. to flush its cache to the HDD, and then block script execution until the cache flush is complete. What is the command sequence for doing this?

**Comment by amotin** [comment, marked as answer]
I am not sure what for would you need it, but `zpool sync ...` should be it.
